src/emailer.js

```diff
@@ -56,17 +56,37 @@ Emailer.registerApp = function (expressApp) {
 
 	// Enable Gmail transport if enabled in ACP
 	if (parseInt(meta.config['email:smtpTransport:enabled'], 10) === 1) {
-		var smtpOptions = {
-			auth: {
+		var smtpOptions = {};
+
+		if(meta.config['email:smtpTransport:user'] || meta.config['email:smtpTransport:pass']) {
+			smtpOptions.auth = {
 				user: meta.config['email:smtpTransport:user'],
-				pass: meta.config['email:smtpTransport:pass'],
-			},
-		};
+				pass: meta.config['email:smtpTransport:pass']
+			};
+		}
 
-		if (meta.config['email:smtpTransport:serice'] === 'nodebb-custom-smtp') {
+		if (meta.config['email:smtpTransport:service'] === 'nodebb-custom-smtp') {
 			smtpOptions.port = meta.config['email:smtpTransport:port'];
 			smtpOptions.host = meta.config['email:smtpTransport:host'];
-			smtpOptions.secure = true;
+
+			switch(meta.config['email:smtpTransport:security']) {
```

Comment:
I'd prefer to just use an if-else if-else here, unless you see room for other options in the future.

Reply:
Changed it to if-else and pushed.